src/index.js

```diff
@@ -1,42 +1,31 @@
 import './style.css';
-
-const scorelist = document.querySelector('.scorelist');
-const names = document.querySelector('.names');
-const numbers = document.querySelector('.numbers');
-const submitbutton = document.querySelector('#submit-button');
-const inputs = document.querySelectorAll('.names, .numbers');
-
-const saveData = (user, data) => localStorage.setItem(user, JSON.stringify(data));
-const getData = (user) => JSON.parse(localStorage.getItem(user));
-
-inputs.forEach((e) => {
-  e.addEventListener('input', () => {
-    const user = {
-      name: names.value,
-      number: numbers.value,
-    };
-    saveData('libraryData', user);
-  });
-});
-
-submitbutton.addEventListener('click', () => {
-  const scores = document.querySelector('.list');
-  scorelist.appendChild(scores);
-
-  const nom = document.createElement('li');
-  const num = document.createElement('li');
-  const Break = document.createElement('br');
-
-  const libraryData = getData('libraryData');
-  if (libraryData !== null) {
-    nom.innerHTML = libraryData.name;
-    num.innerHTML = libraryData.number;
+import { newUser, getScores, newGame } from './myapi.js';
+
+const userValue = document.querySelector('#name');
+const scoreValue = document.querySelector('#score');
+const points = document.querySelector('#points');
+const refresh = document.querySelector('#refresh');
+const submit = document.querySelector('#submit');
+
+submit.addEventListener('click', (e) => {
+  e.preventDefault();
+  if (userValue.value === '' || scoreValue.value === '') {
+    userValue.setCustomValidity('name and score is empty');
+    userValue.reportValidity();
+  } else if (scoreValue.value > 500) {
+    scoreValue.setCustomValidity('highest score is 500');
+    scoreValue.reportValidity();
+  } else {
+    newUser(userValue.value, scoreValue.value);
+    getScores(points);
+    userValue.value = '';
+    scoreValue.value = '';
   }
+});
 
-  scores.appendChild(nom);
-  scores.appendChild(num);
-  scores.appendChild(Break);
-
-  names.value = null;
-  numbers.value = null;
+refresh.addEventListener('click', () => {
+  window.location.reload();
 });
+
+newGame();
```

Comment:
- [OPTIONAL] Basse on the return value of this function call I believe that you don't need that here for your app to run.
You may remove it to keep your code clean and simple.

Reply:
Thank you for your review. required changes applied